lib/QubitCsvImport.class.php

```diff
@@ -101,6 +106,9 @@ public function import($csvFile, $type = null, $csvOrigFileName = null)
     // Figure out whether user option should be added to command
     $commandUser = ($taskClassName == 'csv:import') ? sprintf('--user-id="%s"', sfContext::getInstance()->getUser()->getUserId()) : '';
 
+    // Figure out whether a source name should be added to command
+    $commandSourceName = ($taskClassName != 'csv:authority-relation-import') ? sprintf('--source-name="%s"', escapeshellarg($csvOrigFileName)) : '';
```

Comment:
Why is `source_name` specifically excluded for `csv:authority-relation-import` and no other import types?

Reply:
It wasn't supported when this logic was baked into the main authority import task, but yeah it make be good to add it for consistency and in case it's needed so I'll add it.